photutils/psf/models.py

```diff
@@ -15,7 +24,272 @@
 
 from ..extern.nddata_compat import extract_array
 
-__all__ = ['IntegratedGaussianPRF', 'PRFAdapter', 'prepare_psf_model']
+__all__ = ['Discrete2DModel', 'IntegratedGaussianPRF', 'PRFAdapter',
+           'prepare_psf_model']
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
```

Comment:
Again, remove this line.  It's only for skipping tests using `py.test`.  It does nothing here.